README.md

```diff
@@ -1,4 +1,4 @@
-# PROJECT NAME
+# OOP_SchoolLibrary
 
 > Description the project.
```

Comment:
Kindly update your README file, update the `Built With` section, and the `Authors` section adds your links, and any other section you can customize.